apps/internal/oauth/ops/accesstokens/tokens.go

```diff
@@ -173,14 +174,80 @@ type TokenResponse struct {
 	FamilyID       string                    `json:"foci"`
 	IDToken        IDToken                   `json:"id_token"`
 	ClientInfo     ClientInfo                `json:"client_info"`
-	ExpiresOn      internalTime.DurationTime `json:"expires_in"`
+	ExpiresOn      internalTime.DurationTime `json:"-"`
```

Comment:
DurationTime exists to calculate ExpiresOn from `expires_in`. That doesn't help now that you want to consider a possible value of `expires_on` as well, and you're disabling unmarshaling for this field anyway. So, why not make the type here simply `time.Time`?

Reply:
updated